ko/lessons/basics/pipe-operator.md

```diff
@@ -12,7 +12,7 @@ lang: ko
 
 ## 소개
 
-프로그래밍을 하다보면 코드가 지저분해집니다. 함수의 호출이 따라가기 힘들 정도로 많이 중첩되어 지저분해질 수도 있습니다. 아래 중첩된 함수 호출을 생각해 봅시다.
+프로그래밍을 하다보면 코드가 지저분해질 수 있습니다. 함수의 호출이 따라가기 힘들 정도로 많이 중첩되어 지저분해질 정도로요. 아래 중첩된 함수 호출을 생각해 봅시다.
```

Comment:
Original was:

> Programming can get messy. So messy in fact that function calls can get so embedded that they become very difficult to follow.

I don't think my change is better. but, `can get messy` is not `get messy`. If you have any suggestion, please comment.